docs/plugins/dig.rst

```diff
@@ -143,9 +145,18 @@ Designation options:
 ``clear``
     Clear any designations.
 
-You can also pass a ``-z`` option, which restricts designations to the current
-z-level and down. This is useful when you don't want to designate tiles on the
-same z-levels as your carefully dug fort above.
+Other options:
+
+``--zdown``, ``-d``
+    Only designates tiles on the cursor's z-level and below.
+``--zup``, ``-u``
+    Only designates tiles on the cursor's z-level and above.
+``--cur-zlevel``, ``-z``
+    Only designates tiles on the same z-level as the cursor.
+``--hidden``, ``-h``
+    Allows designation of hidden tiles, and using a hidden tile as the "palette".
```

Comment:
What do you mean by `and using a hidden tile as the "palette"`? I don't understand that part.

Reply:
The selected tile on the cursor which is used to determine what type to dig. I think the word choice is poor on my part but couldn't think of a better word to describe it.

Reply:
oh I see, so it will allow a hidden tile to be selected, and it will mark other tiles based on that tile's material? Is that...desired? If a player has a hidden tile with an unknown (to them) material selected when they run `digtype`, isn't that more likely to be a mistake?

Reply:
It's how the functionality originally was - I do agree that it's probably a mistake but perhaps there's someone out there who really like to live dangerously in that way so I left it in for the hidden mode.